pkg/ddevapp/ddevapp.go

```diff
@@ -939,6 +942,26 @@ func (app *DdevApp) GetAllURLs() []string {
 		}
 		URLs = append(URLs, "http://"+name+httpPort, "https://"+name+httpsPort)
 	}
+
+	dockerIP, err := dockerutil.GetDockerIP()
```

Comment:
Thanks so much for refactoring this correctly. Takes longer to do and is always worth it in a case like this.